shell/common/shell.cc

```diff
@@ -699,6 +699,8 @@ void Shell::OnPlatformViewCreated(std::unique_ptr<Surface> surface) {
   FML_DCHECK(is_setup_);
   FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
 
+  GetIsGpuDisabledSyncSwitch()->SetSwitch(false);
```

Comment:
This isn't how this API was intended to be used.  The setting of the switch was intended to happen when the app loses access to the GPU.  The calls to turn it off and on are matched calls provided by the operating system.  I think this could get really messy fixing our problems this way.

Reply:
I'm going to try to rework this a bit.